display_information/dictionary-renderer-graphics-overlay/README.md

```diff
@@ -1,41 +1,34 @@
-# Dictionary Renderer with Graphics Overlay
+# Dictionary renderer with graphics overlay
 
-Display mil2525d symbols.
+Create graphics using a local mil2525d style file and an XML file with key/value pairs for each graphic.
```

Comment:
this seems quite wordy and too specific for the intro. This probably came from the shared wiki. I'd prefer to see it more succinct, i.e. as it was before "Display mil2525d symbols", the simplest phrasing possible.